template/webapp/serverless.yml

```diff
@@ -0,0 +1,32 @@
+service: TPL_SERVICE_NAME
+
+provider:
+  # general settings
+  name: aws
+  stage: dev
+  region: TPL_REGION
+  profile: default
+  deploymentMethod: direct
+  # global function settings (apply to each declared function)
+  runtime: provided.al2
+  memorySize: 1024
+  timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
```

Comment:
We need to move these back into the function. It's common to start with FPM and add "function" functions, that will not work with this template.